### ISSUE TITLE
demo links in repos should link to the brick-pages example section

### ISSUE THREAD

**Issue by dotch** [mozbrick/brick#230]

**Comment by potch** [comment]
Provided there are demos on the Brick site, yes!

**Comment by dotch** [comment]
yes every component's examples should be living in a subdir of the main pages examples directory.
i will work on that.

**Comment by nicola** [comment]
would we rather have multiple page examples or a single page wrapping them all?

**Comment by fwenzel** [comment]
@dotch should we unassign this or are you planning on still working on this? Thank you!

**Comment by dotch** [comment]
I unassigned from this one, too much work for now!
